README.md

```diff
@@ -13,118 +15,142 @@ An intelligent automation system for creating and monitoring PDP deals on the Fi
 - **Performance Monitoring** - Real-time metrics for deals and retrievals
 - **Add-ons Testing** - FWSS add-ons testing and comparison with provider performance
 - **Analytics Dashboard** - Modern React UI with charts and statistics
-- **Synapse SDK** - Seamless Filecoin storage operations
-- **Smart Contracts** - Automated wallet management and service approvals
+
+## Project Structure
+
+This is a polyrepo containing two separate applications:
+
+```
+dealbot/
+├── backend/          # NestJS API server (Port 8080)
+│   ├── src/
+│   │   ├── deal/            # Deal creation and management
+│   │   ├── retrieval/       # Storage retrieval testing
+│   │   ├── metrics/         # Performance metrics and analytics
+│   │   ├── scheduler/       # Automated task scheduling
+│   │   └── wallet-sdk/      # Wallet and contract operations
+│   └── README.md     # Backend-specific documentation
+└── web/              # React + Vite dashboard (Port 5173)
+    ├── src/
+    └── README.md     # Frontend-specific documentation
+```
 
 ## Quick Start
 
 ### Prerequisites
 
-- Node.js 20+
-- PostgreSQL
-- pnpm
-- Filecoin wallet with tokens
+- **Node.js** 20+
+- **pnpm** (package manager)
+- **PostgreSQL** database
+- **Filecoin wallet** with tokens (for Calibration or Mainnet)
 
-### Installation
+### 1. Clone the Repository
 
 ```bash
-# Clone repository
 git clone https://github.com/FilOzone/dealbot.git
 cd dealbot
+```
 
-# Install dependencies
+### 2. Install Dependencies
+
+```bash
+# Install backend dependencies
+cd backend
 pnpm install
 
-# Configure environment
-cp .env.example .env
-# Edit .env with your wallet and database credentials
+# Install frontend dependencies
+cd ../web
+pnpm install
 ```
```

Comment:
we should use pnpm workspaces and not require users to cd to each project/app to install the deps they need.